### ISSUE TITLE
Engelsk uttale

### ISSUE THREAD

**Issue by anjamaa** [realfagstermer/realfagstermer#860]
Hvorfor er dette en realfagsterm?

--
URI: http://data.ub.uio.no/realfagstermer/c006722
Bruk: http://ub-viz01.uio.no/okapi2/#/search?q=realfagstermer%3A%22Engelsk%20uttale%22
Soksed: https://soksed.biblionaut.net/concepts/REAL006722

**Comment by mittinatten** [comment]
Ja, den burde vi kunne slette, heller bruke tilsvarende fra humord for de titlene?

**Comment by BioHeidi** [comment]
No no. Ikke ennå. Realfagstermer er for bøkene i Realfag(s)biblioteket!

Men dette er en viktig diskusjon som vi nok må ta på et tidspunkt, hvorfor skal vi vedlikeholde parallelle univers, det finnes masse HumSam-termer i Realfagstermer, og masse naturvitenskapelige termer i Humord også.

En sak for mappinggruppa, dette.

**Comment by BioHeidi** [comment]
Og Norsk Generell Tesaurus som NB skal (skulle?) jobbe med.
